Source/Concepts/ConceptMap.cs

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Reflection;
+using Dolittle.SDK.Concepts;
```

Comment:
Cleanup usings?

Reply:
I can do a usings cleanup of the whole solution and push it to master after this is merged